### PR TITLE
Move DTS scripts from meta-dts to this repo

### PR DESCRIPTION
**Pull request by DaniilKl**
Resolves issue https://github.com/Dasharo/dasharo-issues/issues/773.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Makefile for installing/uninstalling scripts.
+
+DTS_PROFILE = dts-profile.sh
```
Comment (macpijan) [inline Makefile:3]: I find this Makefile a bit difficult to read.

It also works not as expected with typical invocation: `make DESTDIR=tmp install` results with:

```
 ls tmp                  
cloud_list  dasharo-deploy  dts  dts-boot  dts-environment.sh  dts-functions.sh  dts-profile.sh  ec_transition
```

Comment (macpijan) [reply Makefile]: I think it was enough to simply port what you had from yocto recipe into Makefile. Please see: c57dc8c60d73ccd0f540148e46be73221ecc67bb

Comment (macpijan) [reply Makefile]:
```
macpijan in ~/projects/github/dasharo/dts-v2 on yocto-cleanup ● λ make DESTDIR=tmp install                                   
install -d tmp/usr/sbin
install -m 0755 include/dts-environment.sh tmp/usr/sbin
install -m 0755 include/dts-functions.sh tmp/usr/sbin
install -m 0755 scripts/cloud_list tmp/usr/sbin
install -m 0755 scripts/dasharo-deploy tmp/usr/sbin
install -m 0755 scripts/dts tmp/usr/sbin
install -m 0755 scripts/dts-boot tmp/usr/sbin
install -m 0755 scripts/ec_transition tmp/usr/sbin
install -m 0755 reports/dasharo-hcl-report tmp/usr/sbin
install -m 0755 reports/touchpad-info tmp/usr/sbin
install -d tmp/etc/profile.d
install -m 0755 dts-profile.sh tmp/etc/profile.d
macpijan in ~/projects/github/dasharo/dts-v2 on yocto-cleanup ● λ tree tmp 
tmp
├── etc
│   └── profile.d
│       └── dts-profile.sh
└── usr
    └── sbin
        ├── cloud_list
        ├── dasharo-deploy
        ├── dasharo-hcl-report
        ├── dts
        ├── dts-boot
        ├── dts-environment.sh
        ├── dts-functions.sh
        ├── ec_transition
        └── touchpad-info

```

Comment (DaniilKl) [reply Makefile]: Ok, if you think so.

Comment (macpijan) [reply Makefile]: @DaniilKl It is less about the structure, more on the functional side. It simply was not working correctly.

**Pull request merged**